src/main/resources/application.properties

```diff
@@ -1,13 +1,23 @@
-server.port = 8099
 
+server.port = 8099
+logging.level.org.hibernate.stat=debug
 #JPA
 spring.jpa.hibernate.ddl-auto=create
 spring.jpa.properties.format_sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
 spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
 spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
-
+#datasource
 spring.datasource.url=jdbc:mysql://localhost:3306/accounting?createDatabaseIfNotExist=true&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=12345678
\ No newline at end of file
+spring.datasource.username=db_test
+spring.datasource.password=12345
+#flyway
+spring.flyway.locations=classpath:db/migration
+spring.flyway.baseline-on-migrate=true
+spring.flyway.validate-on-migrate=true
+spring.flyway.clean-on-validation-error=true
+spring.flyway.user=db_test
+spring.flyway.password=12345
+spring.flyway.schemas=accounting
```

Comment:
AFAIK we don't need schema name, user, and password to make flyway working. The link below should go away as well